leader-election-impl/src/main/java/com/palantir/paxos/PaxosQuorumChecker.java

```diff
@@ -157,7 +157,7 @@ public RESPONSE call() throws Exception {
 
             if (onlyLogOnQuorumFailure && acksRecieved < quorumSize) {
                 for (Pair<String, Throwable> p : toLog) {
-                    log.warn(p.lhSide, p.rhSide);
+                    log.warn("Pair lhSide: {}, Pair rhSide: {}", p.lhSide, p.rhSide);
```

Comment:
Feel this isn't going to be very readable.  The pair is storing <Error message, Throwable> for logging if a certain condition is met.  But for now there is only 1 error message ever stored for the LHS.  Let's just keep a list of the Throwables and write:
```
for (Throwable t : toLog) {
 log.warn(PAXOS_MESSAGE)ERROR, t);
}
```